README.md

```diff
@@ -42,51 +82,88 @@ module "example" {
 | aws\_region | The AWS region to deploy into (e.g. us-east-1). | `string` | `"us-east-1"` | no |
-| subnet\_id | The ID of the AWS subnet to deploy into (e.g. subnet-0123456789abcdef0). | `string` | n/a | yes |
+| bucket\_name | The name of the S3 bucket to publish egress IP address information to. | `string` | n/a | yes |
+| deployment\_role\_arn | The ARN of the IAM role to use to deploy the Lambda and all related resources. | `string` | n/a | yes |
+| domain | The domain hosting the published file(s) containing egress IPs.  Also used for the CloudFront distribution and certificate. | `string` | n/a | yes |
+| ec2\_read\_role\_name | The name of the IAM role that allows read access to the necessary EC2 attributes.  Note that this role must exist in each account that you want to query. | `string` | `"EC2ReadOnly"` | no |
+| extraorg\_account\_ids | A list of AWS account IDs corresponding to "extra" accounts that you want to query for egress IPs to publish. | `list(string)` | `[]` | no |
+| file\_configs | A list of objects that define the files to be published.  "app\_regex" specifies a regular expression matching the application name (based on the var.application\_tag).  "description" is the description of the published file.  "filename" is the name to assign the published file.  "static\_ips" is a list of CIDR blocks that will always be included in the published file.  An example file configuration looks like this: [{"app\_regex": ".*", "description": "This file contains a list of all public IP addresses to be published.", "filename": "all.txt",  "static\_ips": []}, {"app\_regex": "^Vulnerability Scanning$", "description": "This file contains a list of all IPs used for Vulnerability Scanning.", "filename": "vs.txt",  "static\_ips": ["192.168.1.1/32", "192.168.2.2/32"]}] | `list(object({ app_regex = string, description = string, filename = string, static_ips = list(string) }))` | `[]` | no |
+| file\_header | The header template for each published file.  The following variables are available within the template: {domain} - the domain where the published files are located, {filename} - the name of the published file, {timestamp} - the timestamp when the file was published, {description} - the description of the published file | `string` | `"###\n# https://{domain}/{filename}n# {timestamp}\n# {description}\n###\n"` | no |
+| lambda\_function\_description | The description of the Lambda function. | `string` | `"Lambda function to publish egress IP addresses to an S3 bucket configured with a CloudFront distribution for HTTPS access."` | no |
+| lambda\_function\_name | The name of the Lambda function to publish egress IP addresses. | `string` | `"publish-egress-ip"` | no |
+| lambda\_schedule\_interval | The number of minutes betweeen scheduled runs of the Lambda function to publish egress IP addresses. | `string` | `"60"` | no |
```

Comment:
Corresponding suggestions are in `variables.tf` but I think this value should be a `number`. I verified that the existing use in `format()` calls work as written in the Terraform console.

```suggestion
| lambda\_schedule\_interval | The number of minutes betweeen scheduled runs of the Lambda function to publish egress IP addresses. | `string` | `60` | no |
```